src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.NetFx.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using System.Runtime.Versioning;
+
+namespace System
+{
+    public static partial class PlatformDetection
+    {
+        private static string FrameworkName => AppDomain.CurrentDomain.SetupInformation.TargetFrameworkName;
```

Comment:
`AppDomain.CurrentDomain.SetupInformation.TargetFrameworkName` can return null and `FrameworkName` throws when it sees a null.

Reply:
Also an empty string seems to throw as well.

Reply:
If the value you get back is null then we should assume 4.5 -- this is what we do on Desktop today because the attribute with this value was only added in 4.5.